python/ray/rllib/optimizers/aso_minibatch_buffer.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, inqueue, size, num_passes):
         self.inqueue = inqueue
         self.size = size
         self.max_ttl = num_passes
-        self.cur_max_ttl = 1  # ramp up slowly to better mix the input data
+        self.cur_max_ttl = init_num_passes  # 1 for gradual ramp-up, num_passes for immediate minibatching
```

Comment:
can you default this to 1? Leaving a comment like this is very bad - this is not research code

Reply:
@michaelzhiluo